### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by wusixer** [ericmjl/llamabot#3]
Dear @ericmjl, I have tried installing llamabot separately through `pip install llamabot` and through `pyproject.toml` by cloning the repository. In both scenarios, I wasn't able to import the bots correctly. The error message is as following:

```
Python 3.9.7 (default, Sep 16 2021, 08:50:36)
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from llamabot import SimpleBot
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/coxji1/Documents/llamabot/git_repo/llamabot/llamabot/__init__.py", line 12, in <module>
    from .bot import ChatBot, QueryBot, SimpleBot
  File "/Users/coxji1/Documents/llamabot/git_repo/llamabot/llamabot/bot/__init__.py", line 13, in <module>
    from .querybot import QueryBot
  File "/Users/coxji1/Documents/llamabot/git_repo/llamabot/llamabot/bot/querybot.py", line 32, in <module>
    class QueryBot:
  File "/Users/coxji1/Documents/llamabot/git_repo/llamabot/llamabot/bot/querybot.py", line 40, in QueryBot
    doc_paths: List[str] | List[Path] | str | Path = None,
TypeError: unsupported operand type(s) for |: '_GenericAlias' and '_GenericAlias'
```

I wasn't able to find `_GenericAlias` therefore I don't know how to debug it. Could you please help me?

Thanks! Jiayi :)

**Comment by ericmjl** [comment]
Hey @wusixer! 


I think this one is solvable by using Python=3.10. Could you try that?

I'll also update pyproject.toml to reflect that Python 3.10 is the minimum version required.

Specifically, the `|` syntax here for indicating the union of four possible types is the reason for the failure. That was, I think, introduced in py310.

**Comment by wusixer** [comment]
the import issue was solved here

**Comment by ericmjl** [comment]
@wusixer I released llamabot v0.0.64 -- it now comes with a configuration for the default model to use. Please see the tutorial here: https://ericmjl.github.io/llamabot/cli/llamabot/
